learn/contributing/contributing_to_docs.md

```diff
@@ -91,3 +93,75 @@ We ask the following questions:
 7. In the case of new pages, has the page been added in the right location? Should it be linked elsewhere in the documentation?
 
 Nothing makes us happier than getting a thoughtful, helpful PR that saves us time and effort and makes the documentation **even stronger**. Thank you in advance for your contribution!
+
+Our only major requirement for PR contributions is that the author responds to communication requests within a reasonable timeframe.
+
+Once you've opened a PR on this repo, one of our team members will stop by shortly to review it. If your PR is approved, nothing further is required from you. However, **if in seven days you have not responded to a request for further changes or more information, we will consider the PR abandoned and close it**.
+
+If this happens to you and you think there has been some mistake, please let us know and we will try to rectify the situation.
+
+## Local development
+
+### Requirements
+
+- [Node version](https://nodejs.org/en/) >= v14 and <= v16
+
+### Installing and running the docs
+
+```bash
+# Clone the repository
+git clone git@github.com:meilisearch/documentation.git meilisearch-documentation
+
+# Open the newly created directory
+cd meilisearch-documentation
+
+# Install dependencies
+yarn install
+
+# Run Meilisearch documentation on http://localhost:8080
+yarn dev
+```
+
+### Testing
+
+A complete test can be done using the following command:
+
+```bash
+yarn test
+```
+
+The tests are triggered on build and on any pull request to main.
+
+#### Checking dead links
+
+Check for broken or dead links before submitting a pull request using:
+
+```bash
+yarn check-links
+```
+
+#### Checking styling
+
+Run the following command to check for any styling errors like extra spaces or lines in `*.vue`, `*.js`, and `*.md` files:
+
+```bash
```

Comment:
GitHub is messing up the formatting when I try suggesting changes. Can you please fix the formatting for lines 147-149. It should be one code block wrapped in ```